packages/core/src/Stores/Helpers/chart-marker-helpers.js

```diff
@@ -24,7 +25,8 @@ const createMarkerConfig = (marker_type, x, y, content_config) =>
 
 export const getSpotCount = (contract_info, spot_count) => {
```

Comment:
Just wondering...what does this function do ?

Reply:
For tick contracts it create their number. If you buy a `Rise/Fall` (for e.g.) contract then in contract details you'll be able to see a circles with tick's number.
For Accumulators and SmartTrader contracts we should show it as design is different (only tiny grey spots)

Reply:
<img width="711" alt="Screenshot 2023-06-14 at 9 36 55 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/1a6db82b-5de1-4f4e-9192-9cc575f072a1">